### DISCUSSION TITLE
Lombok Integration

### DISCUSSION THREAD

**Discussion by andreachirico79** [burningwave/core#38] [Q&A]
Hi, i noticed that with virtualization off, the compiling procedure ignores che lombok annotation @Data for autogenerating the getter and setter methods. is there a way to inherit this behaviour in some way or do I have to explicit tell to ClassSourceGenerator to add the methods field by field?

thanks

**Comment by Roberto-Gentili** [comment]
> Hi, i noticed that with virtualization off, the compiling procedure ignores che lombok annotation @DaTa for autogenerating the getter and setter methods. is there a way to inherit this behaviour in some way or do I have to explicit tell to ClassSourceGenerator to add the methods field by field?
> 
> thanks

With [this solution](https://github.com/burningwave/core/discussions/35#discussioncomment-8682746) the annotation processing have been disabled: this is the reason why the java compiler do not generate what you except. You should ask for [Stackoverflow](https://stackoverflow.com/search?q=burningwave)

**Reply by andreachirico79** [reply]
thanks!
